pkg/cc/config/loader.go

```diff
@@ -204,6 +200,16 @@ func SetEnvVars(fileContents string) (string, error) {
 	return subEnvVars(fileContents), nil
 }
 
+func filterPaths(paths map[string]bool) []string {
+	var args []string
+	for k := range paths {
+		if k != "" && !strings.Contains(k, "/app") { // append only not empty paths and do not allow /app override
```

Comment:
What exactly does this check for `/app` mean to achieve? Are we saying that we don't allow the substring `/app` to appear in any path used by topazd?